pangeo_forge_recipes/recipes/reference_hdf_zarr.py

```diff
@@ -133,12 +130,19 @@ class HDFReferenceRecipe(BaseRecipe, StorageMixin, FilePatternMixin):
         inlined into the output reference file
     :param output_storage_options: dict of kwargs for creating fsspec
         instance when writing final output
-    :param template_count: the number of occurrences of a URL before it
-        gets made a template. ``None`` to disable templating
-    :param xarray_open_kwargs: kwargs passed to xarray.open_dataset.
-        Only used if `file_pattern` has more than one file.
-    :param xarray_concat_args: kwargs passed to xarray.concat
-        Only used if `file_pattern` has more than one file.
+    :param identical_dims: coordiate-like variables that are assumed to be the
+        same in every input, and so do not need any concatenation
+    :param coo_map: set of "selectors" defining how to fetch the dimension coordinates
+        of any given chunk for each of the concat dimes. By default, this is the variable
+        in the dataset with the same name as the given concat dim, except for "var",
+        where the default is the name of each input variable. See the doc of
+        MultiZarrToZarr for more details on possibilities.
```

Comment:
Can we get intersphinx going on kerchunk so we can cross-reference its documentation here?

Reply:
What do I need to do?